src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -917,6 +917,11 @@ static int do_list_mapped_devices()
 
 static int parse_args(vector<const char*>& args, std::ostream *err_msg, Config *cfg)
 {
+  std::string conf_file_list;
+  std::string cluster;
+  CephInitParameters iparams = ceph_argparse_early_args(
+          args, CEPH_ENTITY_TYPE_CLIENT, &cluster, &conf_file_list);
```

Comment:
@liupan1111 It would be nice to pass the results of `ceph_argparse_early_args` to config, i.e:

1) set `config.name = iparams.name`
2) set `config.cluster = cluster`
3) if `conf_file_list` is not empty, pass it to `config.parse_config_files` instead of `nullptr`.

This should make "rbd_default_pool" config option to be used properly when it is changed in config.

Reply:
cool!